PRIVACY_POLICY.md

```diff
@@ -0,0 +1,65 @@
+# Swarm-Extension Privacy Policy
+
+The Swarm Extension belongs to the Swarm Foundation with the purpose of offering web3 environment
+for running decentralised applications (dApps). It means, it provides similar functionalities as web2 browsers,
+so sensitive information may be handled by the browser-extension such as Bee client configuration
+or cross-domain localStore. Although the extension handles these sensitive information for
+the basic security and functionality of dApps, the Swarm Foundation does not share any of these data
+with other 3rd parties, nor collect it.
+
+By installing Swarm Extension, you accept this privacy policy.
+
+## Do we collect any information from you?
+
+We do not collect any data from you, including personal data.
+
+## Sharing Data By The Help of The Extension
+
```

Comment:
The Swarm Extension can be configured to connect to an Ethereum Swarm Bee client node (which is a peer-to-peer (P2P) storage client).

should/can we use the word Ethereum in the text?